### PR TITLE
Documenting /account/settings endpoint (partially)

### PR DESCRIPTION
**Pull request by fentie**
This is only partially complete but it covers the data the mobile devs need for now w/o making things too ugly.

Refs ACORE-553

### CODE REVIEW

File: source/methods/accounts.md.erb [hunk 1]
```diff
@@ -79,6 +79,54 @@ If the account being queried for is a primary account, the response provides an
...
+
+### HTTP Request
+`POST <%=@api_prefix%>/2/account/settings`
+
+> Example Request
+
+```shell
+curl -X POST <%=@api_prefix%>/2/account/settings \
+ -H "W-Token: <%=@wiw_token%>" \
+ -data '{"company":"123 Company","ref_employees":7,"subdmain":"123-company","industry_id":14}'
```
Comment (bvisness) [inline source/methods/accounts.md.erb]: Typo: `"subdmain"`

**Comment by fentie** [comment]
@bvisness I think that typo got fixed. Can this be merged? our IOS devs want to use this this sprint

**Comment by bvisness** [comment]
@fentie Deployed. I noticed after deploying, though, that the example request and response were in the wrong place. (I really should have checked this out for myself!) That is fixed in commit da10e9d11ef8e21abab16fee87e29d6692f16608.

Basically, just don't forget that the request and response have to be the first things in each section, or they get pushed down the page by the other content.

**Comment by fentie** [comment]
Ah cool, thanks. I probably started copying & pasting the block at the wrong spot but it still had all the right sections. I'll try to keep an eye out for that next time.

**Pull request merged**